src/lib/components/search/display/images/ShowMore.svelte

```diff
@@ -16,20 +16,33 @@
 	let pages: number = 2;
 
 	// functions
-	const showMore = async () => {
-		const [newOffset, newResults]: [number, ResultType[]] = await fetchResultsLazily(query, start, offset, pages, results, paramsString);
+	async function handleSubmit() {
+		const [newOffset, newResults] = await fetchResultsLazily(
+			query,
+			start,
+			offset,
+			pages,
+			results,
+			paramsString
+		);
 		offset = newOffset;
 		results = newResults;
-	};
+	}
 </script>
 
-<!-- TODO: revert back to form for no-JS users but use enhancing to load the results lazily like below -->
-<div class="pb-5 flex flex-row justify-center">
+<form
+	on:submit|preventDefault={handleSubmit}
+	method="get"
+	action="/search"
+	class="pb-5 flex flex-row justify-center"
+>
+	<input type="hidden" name="q" value={query} />
+	<input type="hidden" name="start" value={start + offset} />
+	<input type="hidden" name="pages" value={pages} />
 	<button
-		on:click={showMore}
-		type="button"
-		class="mx-auto mb-5 h-12 w-1/5 rounded-lg overflow-hidden dark:bg-zinc-800 hover:dark:bg-zinc-700 hover:ring-2 hover:ring-hearchco-primary hover:dark:ring-hearchco-secondary text-zinc-500 dark:text-zinc-200 hover:text-hearchco-primary hover:dark:text-hearchco-secondary shadow-sm dark:shadow-none border border-gray-100 dark:border-0 duration-200 ease-in-out"
+		type="submit"
+		class="mx-auto mb-5 h-12 w-1/2 sm:w-1/3 lg:w-1/4 2xl:w-1/5 rounded-lg overflow-hidden dark:bg-zinc-800 hover:dark:bg-zinc-700 hover:ring-2 hover:ring-hearchco-primary hover:dark:ring-hearchco-secondary text-zinc-500 dark:text-zinc-200 hover:text-hearchco-primary hover:dark:text-hearchco-secondary shadow-sm dark:shadow-none border border-gray-100 dark:border-0 duration-200 ease-in-out"
```

Comment:
How about doing `w-[80dvw]` here instead of `w-1/2`? It seems to me that this arbitrary value might behave better when it comes to really narrow devices (e.g. samsung fold)?